extensions/amp-analytics/0.1/test/test-vendors-new.js

```diff
@@ -104,6 +109,34 @@ describes.realWin(
               config = analytics.config_;
               done();
             });
+
+            // Have to get service after analytics element is created
+            const variableService = variableServiceForDoc(doc);
+
+            window.sandbox
+              .stub(variableService, 'getMacros')
+              .callsFake(function() {
+                // Add all the macros in amp-analytics
+                const merged = {...this.macros_, ...elementMacros};
+
+                // Change the resolving function
+                const keys = Object.keys(merged);
+                for (let i = 0; i < keys.length; i++) {
+                  const key = keys[i];
+                  // Three opt params here because that is currently
+                  // the max number of params for any macro
+                  merged[key] = (opt_param, opt_param2, opt_param3) => {
```

Comment:
can we use `(...args) =>` to support any number of arguments?

Reply:
Yup, good call :)